examples/ruby/ruby_client.rb

```
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+# Load the gem
+require 'openapi_client'
+
+CERT_DIR = 'config/https'
+SSL_CERT_FILE = 'ca.crt'
+CONJUR_CERT_FILE = 'conjur.crt'
+CONJUR_KEY_FILE = 'conjur.key'
+
+ACCOUNT="dev"
+LOGIN="admin"
+ADMIN_API_KEY=ENV["CONJUR_ADMIN_API_KEY"]
```

We should probably throw an error if this env variable isn't set